src/test/integrationSuiteSetup.ts

```diff
@@ -18,4 +18,5 @@ beforeEach(async () => {
 
 afterEach(async () => {
   await cleanupDatabaseForCurrentWorker();
+  jest.restoreAllMocks();
```

Comment:
I agree that mocks should be isolated to the test, but I would have thought jest should do it by default instead of needing the explicit call. Hmm.

Reply:
Hmm indeed :(

Reply:
Good enough for now.